### PR TITLE
docker-hub Admin right gives access to build logs

### PR DESCRIPTION
**Pull request by tristan0x**

### CODE REVIEW

**Comment by unclejack** [comment]
@tristan0x Please squash your commits into one and force push to this branch.

**Comment by tristan0x** [comment]
@unclejack it's done!

**Comment by SvenDowideit** [comment]
LGTM - @moxiegirl

File: docs/docker-hub/accounts.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ repository access rights to groups:
 - `Write` access users are able to push to non-automated repositories on the Docker
   Hub.
 - `Admin` access allows the user to modify the repositories "Description", "Collaborators" rights,
-  "Mark as unlisted", "Public/Private" status and "Delete".
+  "Mark as unlisted", "Public/Private" status and "Delete", and to retrieve build logs.
```
Comment (moxiegirl) [inline docs/docker-hub/accounts.md]: "and to retrieve build logs" did you mean:

"Mark as unlisted", "Public/Private" status, and "Delete" to retrieve build logs.

Comment (tristan0x) [inline docs/docker-hub/accounts.md]: No.
"Automated build" repositories trigger a new build of the Docker image whenever a change occurs in the tracked repository. Every build is recorded and visible through the *builds history* page for users with Read permission. But it is required to have Admin permission to actually read a build log.

Comment (moxiegirl) [inline docs/docker-hub/accounts.md]: Thanks for the clarification. This section on rights has some additional structural issues. If you don't mind, let's clarify them all.  If you do mind, happy to carry.

----> https://gist.github.com/moxiegirl/1c34957b9edc0f6d528c

### Repository group permissions

Use organization groups to manage the users that can interact with your repositories.

You must be in an organization's `Owners` group to create a new group, Hub
repository, or automated build. As an `Owner`, you then delegate the following
repository access rights to groups:

| Access Right | Description                                                                                                                                                                                                                                                                |
|--------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `Read`       | Users with this right can view, search, and pull a private repository.                                                                                                                                                                                                     |
| `Write`      | Users with this right can push to non-automated repositories on the Docker Hub.                                                                                                                                                                                            |
| `Admin`      | Users with this right can modify a repository's "Description", "Collaborators" rights. They can also mark a repository as unlisted, change its  "Public/Private" status and "Delete" the repository. Finally, `Admin` rights are required to read the build log on a repo. |
|              |                                                                                                                                                                                                                                                                            |

Regardless of their actual access rights, users with unverified email addresses
have `Read` access to the repository. Once they have verified their address,
they have their full access rights as granted on the organization.

**Comment by thaJeztah** [comment]
@tristan0x do you want do update your PR with the changes that @moxiegirl suggested, or do you want us to carry?

**Comment by tristan0x** [comment]
Hi Sebastiaan,
I'll update it today.
Regards

**Comment by thaJeztah** [comment]
@tristan0x thanks!

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Comment by moxiegirl** [comment]
cherry-picked

**Pull request merged**